lib/phoenix/digester.ex

```diff
@@ -83,15 +83,21 @@ defmodule Phoenix.Digester do
     write_manifest(latest, digests, output_path)
   end
 
+  @comment "This is file was auto-generated by `mix phx.digest`. Remove it and all generated artefacts with `mix phx.digest.clean --all`"
```

Comment:
There is a typo there. 

```suggestion
  @comment "This file was auto-generated by `mix phx.digest`. Remove it and all generated artefacts with `mix phx.digest.clean --all`"
```